hugo/content/docs/persistence.md

```diff
@@ -102,4 +101,6 @@ After applying any change on actor's state, the second delegate is used and the
 
 ## Persistence using events and snapshots
 
-If actor is using eventsourcing to maintain its state then it is possible to [Proto.Persistence module](persistence-proto-persistence.md) to make it easier.
+If actor is using eventsourcing to maintain its state then it is possible to use [Proto.Persistence module](persistence-proto-persistence.md) to make it easier.
```

Comment:
I this intentionally part of the PR?

Reply:
Yes, to fix the broken last paragraph link issue.